src/ontology/ro-edit.owl

```diff
@@ -3933,6 +3935,7 @@ ObjectPropertyRange(obo:RO_0002437 obo:BFO_0000040)
 
 # Object Property: obo:RO_0002438 (trophically interacts with)
 
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0002438 "lions trophically interact with the zebras that they eat")
```

Comment:
@diatomsRcool you added the example in `trophically interacts with` instead of `trophic parasite of`, which is the new object property.

Reply:
I don't understand this comment. the object property is trophically interacts with and that's what it says in the example

Reply:
In this PR, you created two object properties, `trophic parasite of` and `symbiotrophically interacts with`, and we're asking for an example of usage of these properties. Just so you know, you don't need to remove the ones you have added today, as they were missing before.

Reply:
ok, I think I dealt with it